docs/technical_reference/property_models/activated_sludge_model.rst

```diff
@@ -0,0 +1,144 @@
+   "Oxygen half-saturation coefficient for autotrophic biomass, K_O,A", ":math:`K_{O,A}`", "K_OA", "[p]", 0.0004, ":math:`\text{kg -COD/}\text{m}^{3}`"
+   "Nitrate half-saturation coefficient for denitrifying heterotrophic biomass, K_NO", ":math:`K_{NO}`", "K_NO", "[p]", 0.0005, ":math:`\text{kg NO}_{3}\text{-N/}\text{m}^{3}`"
+   "Decay coefficient for heterotrophic biomass, b_H", ":math:`b_H`", "b_H", "[p]", 0.3, ":math:`\text{d}^{-1}`"
+   "Decay coefficient for autotrophic biomass, b_A", ":math:`b_A`", "b_A", "[p]", 0.05, ":math:`\text{d}^{-1}`"
+   "Correction factor for mu_H under anoxic conditions, eta_g", ":math:`η_g`", "eta_g", "[p]", 0.8, ":math:`\text{dimensionless}`"
+   "Correction factor for hydrolysis under anoxic conditions, eta_h", ":math:`η_h`", "eta_h", "[p]", 0.8, ":math:`\text{dimensionless}`"
+   "Maximum specific hydrolysis rate, k_h", ":math:`k_h`", "k_h", "[p]", 3.0, ":math:`\text{d}^{-1}`"
+   "Half-saturation coefficient for hydrolysis of slowly biodegradable substrate, K_X", ":math:`K_X`", "K_X", "[p]", 0.1, ":math:`\text{dimensionless}`"
+   "Ammonia Half-saturation coefficient for autotrophic biomass, K_NH", ":math:`K_{NH}`", "K_NH", "[p]", 1.0, ":math:`\text{kg NH}_{3}\text{-N/}\text{m}^{3}`"
+   "Ammonification rate, k_a", ":math:`k_a`", "k_a", "[p]", 0.00005, ":math:`\text{m}^{3}\text{/}\text{kg COD . d}`"
+
+
+Properties
+----------
+.. csv-table::
+  :header: "Description", "Symbol", "Variable", "Index", "Units"
+
+  "Fluid specific heat capacity", ":math:`c_p`", "cp", "None", ":math:`\text{J/kg/K}`"
+  "Mass density", ":math:`\rho`", "dens_mass", "[p]", ":math:`\text{kg/}\text{m}^3`"
+  "Soluble inert organic matter, S_I", ":math:`S_I`", "S_I", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Readily biodegradable substrate S_S", ":math:`S_S`", "S_S", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Particulate inert organic matter, X_I", ":math:`X_I`", "X_I", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Slowly biodegradable substrate X_S", ":math:`X_S`", "X_S", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Active heterotrophic biomass X_B,H", ":math:`X_{B,H}`", "X_BH", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Active autotrophic biomass X_B,A", ":math:`X_{B,A}`", "X_BA", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Particulate products arising from biomass decay, X_P", ":math:`X_P`", "X_P", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Oxygen, S_O", ":math:`S_O`", "S_O", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Nitrate and nitrite nitrogen, S_NO", ":math:`S_{NO}`", "S_NO", "[p]", ":math:`\text{kg/m}^{3}`"
+  "NH4 :math:`^{+}` + NH :math:`_{3}` Nitrogen, S_NH", ":math:`S_{NH}`", "S_NH", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Soluble biodegradable organic nitrogen, S_ND", ":math:`S_{ND}`", "S_ND", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Particulate biodegradable organic nitrogen, X_ND", ":math:`X_{ND}`", "X_ND", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Alkalinity, S_ALK", ":math:`S_{ALK}`", "S_ALK", "[p]", ":math:`\text{kmol HCO}_{3}^{-}\text{/m}^{3}`"
```

Comment:
Based on how the property model is formulated, these are listed as components, while calling `conc_mass_comp['S_I"]`  would give us the mass concentration of soluble inert organic matter (for example). 

Maybe we'd want to move these to a separate "Components" section then, right after "Sets" and eliminate the Index and Units columns.